pkg/api/defaults.go

```diff
@@ -138,7 +138,7 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpdate bool) {
 				o.KubernetesConfig.MobyVersion = DefaultMobyVersion
 			}
 		case Containerd, ClearContainers, KataContainers:
-			if o.KubernetesConfig.ContainerdVersion == "" {
+			if o.KubernetesConfig.ContainerdVersion == "" || isUpdate {
```

Comment:
Can we get unit tests for this as well?

Reply:
it's already tested

Reply:
oh wait let me rebase and fix the log messages